lib/doorkeeper/orm/active_record/application.rb

```diff
@@ -31,6 +32,18 @@ def self.authorized_for(resource_owner)
       where(id: resource_access_tokens.select(:application_id).distinct)
     end
 
+    # Fallback to existing, default behaviour of assuming all apps to be
+    # confidential if the migration hasn't been run
+    def confidential
+      return super if self.class.column_names.include?('confidentialz')
```

Comment:
Is this a typo?

Reply:
Yeah. I goofed up there. Resolved it and so tests should pass